package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
 	 "start": "heroku-node-settings index.js",
 	 "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=node_modules --load-path=node_modules/@financial-times --load-path=. | postcss --use autoprefixer --output public/main.css --map",
-	 "build:js": "mkdir -p public && node_modules/scrumple/scrumple ./src/main.js | swc -f - -C minify > public/main.js",
+	 "build:js": "mkdir -p public && esbuild --bundle --minify  ./src/main.js > public/main.js",
```

Comment:
question: why was the swc command also removed? If it was intentional then we can also remove the two dev-dependencies: "@swc/cli" and "@swc/core"

Reply:
```suggestion
	 "build:js": "mkdir -p public && esbuild --bundle ./src/main.js | swc -f - -C minify > public/main.js",
```